### ISSUE TITLE
weighted sum confusion

### ISSUE THREAD

**Issue by ifmaq1** [linjieli222/VQA_ReGAT#36]
I want to work on vqa2 dataset 

Could you please explain how did you implement the stated line in code.
**Our best results are achieved by combining the best single relation models through weighted sum**

I mean how do we combine all the models while evaluating using 
**python3 eval.py --output_folder pretrained_models/regat_implicit/ban_1_implicit_vqa_196**

Also, please let me know did you use BAN model in place of butd for best results when using the weighted sum?
**python3 main.py --config config/butd_vqa.json**

**Comment by linjieli222** [comment]
Hi there, 

Thanks for your interests in our project.
The eval.py did not include the weighted sum part. As shown in equation 10 of the paper,  the weighted sum is applied at prediction level. You can first generate the predictions and write a simple weighted sum function to aggregate the logits for each question. 

I assume that you are referring to Table 3 results, the best results are obtained by only using BAN with the three types of relations. Results on other fusion methods such as BUTD and MUTAN are shown in Table 1.

**Comment by ifmaq1** [comment]
@linjieli222 thank you for your response. Yes I'm referring to Table 3 results. Did you use

**python3 main.py --config config/ban_vqa.json** for training 

**and later on, evaluated the three graph attentions with BAN** 
python3 eval.py --output_folder pretrained_models/regat_implicit/ban_1_implicit_vqa_196
python3 eval.py --output_folder pretrained_models/regat_implicit/ban_1_semantic_vqa_7971
python3 eval.py --output_folder pretrained_models/regat_implicit/ban_1_spatial_vqa_1687

then answer Predictions are generated through them, and I need to apply equation 10!**

**Am I right?**
